mysql-test/suite/versioning/r/alter.result

```diff
@@ -76,13 +76,17 @@ t	CREATE TABLE `t` (
   `a` int(11) DEFAULT NULL
 ) ENGINE=MyISAM DEFAULT CHARSET=latin1
 alter table t add column trx_start timestamp(6) as row start;
-ERROR HY000: Table `t` is not system-versioned
```

Comment:
Should in this place `--error ER_VERS_NOT_VERSIONED` be used, since we drop system versioning before?

Reply:
I don't think so, because adding system versioning will not help command to succeed.

Reply:
Ohh , I have read wrong error message, thought it was only about sys-versioning.